### DISCUSSION TITLE
Selecting iCal - fields to be shown

### DISCUSSION THREAD

**Discussion by Sailor-Fhem** [niccokunzmann/open-web-calendar#501] [Q&A]
The calender app as standalone on the RasPi works like a charm! 

The iCal (.ics) file I am using contains a lot of nonsense Information which are indicated on the OenWebCalendar.
Therefore it pushes away the important information.

Example calendar.ics:

```
BEGIN:VEVENT
CLASS:PUBLIC
STATUS:CONFIRMED
TRANSP:TRANSPARENT
CATEGORIES:EDUCATION
URL:https://mese.webuntis.com
UID:1029823
LOCATION:Sporthalle groß
DTSTART;TZID=Europe/Berlin:20241021T073000
DTEND;TZID=Europe/Berlin:20241021T081000
DTSTAMP;TZID=Europe/Berlin:20241012T133459
SUMMARY:9.1 Sport Kumlehn
DESCRIPTION:Klasse: 9.1\nUnterricht: Sport\nOrt: Sporthalle groß\nLehrkraft: Kumlehn
END:VEVENT 
```
The indication on the calendar is
`08:10 | EDUCATION |`

What I want to see without moving the mouse over is the content of "SUMMARY"
`9.1 Sport Kumlehn`

Is there a possibility to choose the field to be indicated?

Sailor

**Comment by niccokunzmann** [comment]
Could you open this as an issue? At the moment, it is not possible to choose that.
A screenshot would be nice, too.
